### DISCUSSION TITLE
Usage with Vite

### DISCUSSION THREAD

**Discussion by ccollie** [reaviz/reaviz#43] [General]
Has anyone tried this with Vite (which uses `rollup`) ? I'm getting an error `countUp.min.js:1 Uncaught ReferenceError: require is not defined`

**Comment by amcdnl** [comment]
interesting - can you post your config? I use rollup to package this so its odd that doesn't work.

**Reply by ccollie** [reply]
`vite` wraps `rollup` under the hood, so I don't have a `rollup` specific config. My guess from the error is that there is a config issue in handling `commonjs` imports, which is strange since `vite` handles that automatically. 
 I'll try to set explicit handling using `rollup` and see what happens.
